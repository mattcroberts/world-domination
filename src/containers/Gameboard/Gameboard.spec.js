import { shallow } from 'enzyme';
import React from 'react';

import { GameboardContainer } from './index';

describe('GameboardContainer', () => {
    const nations = {};
    it('should render', () => {
        const component = shallow(
            <GameboardContainer
                nations={nations}
                nationClick={() => undefined}
            />
        );

        expect(component).toMatchSnapshot();
    });
});
