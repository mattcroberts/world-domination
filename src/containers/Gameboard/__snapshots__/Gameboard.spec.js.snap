// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameboardContainer should render 1`] = `
<GameboardWrapper
  nationClick={[Function]}
  nations={Object {}}
/>
`;
